trojsten/rules/test.py

```diff
@@ -159,19 +159,23 @@ def test_year_only(self):
 
     def test_camps_only(self):
         # Coefficient = 3: year = 1, successful semesters = 2, mo = 0
-        EventParticipant.objects.create(
-            event=self.camps[5], user=self.test_user, type=EventParticipant.PARTICIPANT, going=True
-        )
-        EventParticipant.objects.create(
-            event=self.camps[4], user=self.test_user, type=EventParticipant.PARTICIPANT, going=True
-        )
+        for i in range(3, 6):
```

Comment:
Toto mi trošku nesedí, vyrobí to 3 účasti na sústredení, ale úspešné semestre majú byť len dva?

Tiež by som otestoval, že sa započíta náhradník (pokojne v tomto teste).

Reply:
Tam je problém s tým že nechceš rátať pri výpočte koeficientu aktuálneho semestra sústredenia spojené s aktuálnym semestrom. Inak by ti vo výsledkovke mohli ľudia preskočiť z alfy do bety po pozvaní na sústredenie. 

Prvé dva sústredenia boli pred aktuálnym semestrom, teda sa počítajú, ale tretí nechceš započítať.

Reply:
Aha, ten posledný camp je príliš neskoro. Tak potom je to v poriadku.